services/core-web/src/components/common/DocumentTable.tsx

```diff
@@ -70,6 +87,19 @@ export const DocumentTable = ({
   openDocument,
   ...props
 }: DocumentTableProps) => {
+  const [rowSelection, setRowSelection] = useState([]);
+  const [isBeginCompressionModalVisible, setBeginCompressionModalVisible] = useState(false);
+  const [isReadyForDownloadModalVisible, setReadyForDownloadModalVisible] = useState(false);
+  const [isCompressionProgressVisible, setCompressionProgressVisible] = useState(false);
+  const [compressionProgress, setCompressionProgress] = useState(0);
+  const [documentTypeCode, setDocumentTypeCode] = useState("");
+  const [notificationTopPosition, setNotificationTopPosition] = useState(0);
+  const [fileToDownload, setFileToDownload] = useState("");
+  const [projectTitle, setProjectTitle] = useState("");
```

Comment:
Instead of the project title, can we use "entity title" or something that can apply to any document?

Reply:
This has been adjusted to entityTitle now